ompi/mca/coll/adapt/coll_adapt_component.c

```diff
@@ -81,15 +78,34 @@ mca_coll_adapt_component_t mca_coll_adapt_component = {
 /* Open the component */
 static int adapt_open(void)
 {
+    int param;
+    mca_coll_adapt_component_t *cs = &mca_coll_adapt_component;
+
+    /*
+     * Get the global coll verbosity: it will be ours
+     */
+    param = mca_base_var_find("ompi", "coll", "base", "verbose");
```

Comment:
I think this is already available in the coll base component, and you should be able to access it directly using `ompi_coll_base_framework.framework_verbose`.